src/shared/components/select/OptimizedOption.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { components } from 'react-select';
+
+export default class OptimizedOption extends React.Component<any> {
+    public render() {
+        delete this.props.innerProps.onMouseMove;
```

Comment:
curious what this is for

Reply:
@alisman This is because the mouse move event and mouseover event will force the react select component to iterate the whole list of options, it will not impact loading speed, but when you move between options, you will notice significant lagging doing so.
And since these two events are not very important, we can just remove those to get a speedy response.